schema/resources/assertion.json

```diff
@@ -0,0 +1,19 @@
+{
+    "title": "arxiv-external-links",
+    "description": "Clearinghouse for relations between arXiv e-prints and external resources",
```

Comment:
Same for description -- specific to the resource type

Reply:
0f64e45